sql/testdata/builtin_function

```diff
@@ -22,7 +22,7 @@ SELECT LOWER('RoacH7')
 LOWER('RoacH7')
 roach7
 
-statement error LOWER: argument type mismatch: string expected, but found int
+statement error unknown signature for LOWER: LOWER\(int\)
 SELECT LOWER(32)
 
 query T
```

Comment:
can you add some for CONCAT (and ideally `random()` to test out the diff between zero args and variadic w/o args)?